pkg/flowaggregator/flowaggregator.go

```diff
@@ -342,8 +346,9 @@ func (fa *flowAggregator) initExportingProcess() error {
 			CollectorProtocol:   fa.externalFlowCollectorProto,
 			ObservationDomainID: fa.observationDomainID,
 			TempRefTimeout:      1800,
-			PathMTU:             0,
+			PathMTU:             1400,
```

Comment:
is this a related change? could you add a comment?

Reply:
It's not related. Have it removed. Thanks!